Source/WebCore/rendering/RenderLayer.cpp

```diff
@@ -5915,6 +5915,10 @@ static bool hasVisibleBoxDecorationsOrBackground(const RenderElement& renderer)
 #if HAVE(SUPPORT_HDR_DISPLAY)
 static bool rendererHasHDRContent(const RenderElement& renderer)
 {
+    auto& style = renderer.style();
+    if (style.dynamicRangeLimit().toPlatformDynamicRangeLimit() == PlatformDynamicRangeLimit::standard())
```

Comment:
We shouldn't have to convert to a `PlatformDynamicRangeLimit` to ask if it's `standard`.

Reply:
Fixed.